elisp-bytecode.texi

```diff
@@ -1320,6 +1515,22 @@ PC  Byte  Instruction
  1  135   return
 @end verbatim
 
+@node stack-set
+@unnumberedsubsec @code{stack-set} (180)
+@kindex stack-set
+Like discard. [What's the difference?[
+
+Added in Emacs 24.1
+
+@node stack-set2
+@unnumberedsubsec @code{stack-setN} (181)
+@kindex stack-set2
+Like discardN. [What's the difference?]
```

Comment:
I don't think that's true. stack-set discards only one value, after copying it to a position further up the stack; however, stack-set[0] is indeed equivalent to discard.